ui/app/components/job-editor.js

```diff
@@ -12,12 +12,13 @@ export default class JobEditor extends Component {
 
   @tracked error = null;
   @tracked planOutput = null;
-  @tracked view = 'job-spec';
   @tracked isEditing;
+  @tracked view;
 
   constructor() {
     super(...arguments);
     this.isEditing = !!(this.args.context === 'new');
+    this.view = this.args.specification ? 'job-spec' : 'full-definition';
```

Comment:
Question: is this also accomplishable by having `view` be a getter with this logic embedded? Is there a particular reason to do it in a constructor instead?